frontend.py

```diff
@@ -20,16 +29,34 @@
 def my_form():
     username = request.args.get('username', '')
     if username:
-        return render_template('status.html', username=username, sites=sites)
+        return render_template('status.html',
+                               username=username,
+                               sites=sites,
+                               host_backend=HOST_BACKEND,
+                               port_backend=PORT_BACKEND)
     return render_template('form.html')
 
 
 @app.route('/', methods=['POST'])
 @cross_origin(origin='*')
 def my_form_post():
     username = request.form['text']
-    return render_template('status.html', username=username, sites=sites)
+    return render_template('status.html',
+                           username=username,
+                           sites=sites,
+                           host_backend=HOST_BACKEND,
+                           port_backend=PORT_BACKEND)
 
 
 if __name__ == '__main__':
+    handler = logging.FileHandler('static/logs.log')
+    formatter = logging.Formatter('[%(levelname)s|%(filename)s:%(lineno)s] '
+                                  '%(asctime)s %(message)s')
+    handler.setLevel(logging.INFO)
+    handler.setFormatter(formatter)
+    app.logger.addHandler(handler)
+    if not (HOST_BACKEND and PORT_BACKEND):
+        app.logger.error('Both HOST_BACKEND and PORT_BACKEND'
+                         ' should be set before running')
+        sys.exit(1)
     app.run(host='0.0.0.0', port=8522)
```

Comment:
I think we can let host be `0.0.0.0` since that's how it is when run deployed on ``heroku`` but the ports for both frontend and backend also needs to be in ``.env`` and reflected here as well as in ``username_api.py``

Reply:
I think it is good apart from this change.

Reply:
I think host and port for `frontend.py` doesn't need 
since `frontend.py` depends on `username_api.py` 
but `username_api.py` doesn't depends on `frontend.py`, it's independent.

I can't understand why we should add port for frontend.

Reply:
because port is a hard coded value here. ``8522`` is very random which I came up with. 

It belongs to ``.env`` and not in the code.

Reply:
ok